core/meterer/util.go

```diff
@@ -0,0 +1,164 @@
+package meterer
+
+import (
+	"context"
+	"math/big"
+
+	commonaws "github.com/Layr-Labs/eigenda/common/aws"
+	test_utils "github.com/Layr-Labs/eigenda/common/aws/dynamodb/utils"
+	"github.com/Layr-Labs/eigenda/core"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+)
+
+func DummyCommitment() core.G1Point {
+	commitment := core.NewG1Point(big.NewInt(123), big.NewInt(456))
+	return *commitment
+}
+
+func CreateReservationTable(clientConfig commonaws.ClientConfig, tableName string) error {
+	ctx := context.Background()
+	_, err := test_utils.CreateTable(ctx, clientConfig, tableName, &dynamodb.CreateTableInput{
+		AttributeDefinitions: []types.AttributeDefinition{
+			{
+				AttributeName: aws.String("AccountID"),
+				AttributeType: types.ScalarAttributeTypeS,
+			},
+			{
+				AttributeName: aws.String("BinIndex"),
+				AttributeType: types.ScalarAttributeTypeN,
+			},
+		},
+		KeySchema: []types.KeySchemaElement{
+			{
+				AttributeName: aws.String("AccountID"),
+				KeyType:       types.KeyTypeHash,
+			},
+			{
+				AttributeName: aws.String("BinIndex"),
+				KeyType:       types.KeyTypeRange,
+			},
+		},
+		GlobalSecondaryIndexes: []types.GlobalSecondaryIndex{
+			{
+				IndexName: aws.String("AccountIDIndex"),
```

Comment:
Do we need this index? Looks like we already key by `AccountID`

Reply:
During testing, I found that this IndexName was needed to enable `QueryIndex`. I referenced the pattern like [here] with `BlobStatus` key and `StatusIndex` index (https://github.com/Layr-Labs/eigenda/blob/35e0b6b11182f1ab0fbbc7453f515b78703d9b26/disperser/common/blobstore/blob_metadata_store.go#L462).

Reply:
Yeah if you need to query by something other than the table's default keys, then building indexes on those attributes may help. However, looks like we can query by the table's default keys, in which case we can query from the table [directly](https://github.com/Layr-Labs/eigenda/blob/35e0b6b11182f1ab0fbbc7453f515b78703d9b26/common/aws/dynamodb/client.go#L195)